core/git-utils/index.js

```diff
@@ -209,6 +209,41 @@ function hasCommit(opts) {
   return retVal;
 }
 
+function fetchGitRemote(opts) {
+  log.silly("fetchGitRemote");
+  ChildProcessUtilities.execSync("git", ["remote", "update"], opts);
```

Comment:
Hah, 10+ years using git, and I've literally never used this `git remote` subcommand. Sweet!